### ISSUE TITLE
Disable the depth test when clamped to ground zooms to wrong location

### ISSUE THREAD

**Issue by mramato** [AnalyticalGraphicsInc/cesium#5957]
Select `Disable the depth test when clamped to ground` in http://localhost:8080/Apps/Sandcastle/index.html?src=Billboards.html&label=All and you'll be looking at the size of a mountain.  You need to move the camera to see the billboard.

This is using the `viewer.zoomTo`, so maybe we introduced a bug when clamping to ground?

**Comment by IanLilleyT** [comment]
Yup, noticed this too. I think what's happening is the billboard is initially clamping to a low LOD tile and the camera zooms to that spot. Then higher LOD tiles load in and the entity shifts upwards, so now the camera is underground looking at nothing.

**Comment by IanLilleyT** [comment]
I ran into this problem again during a CesiumJS release. My previous comment is not the full story, because even if the camera tracks the entity it still gets lodged underground.

I created a more minimal example below:

https://sandcastle.cesium.com/#c=bZJvb9MwEMa/yimvUik4y2CIla6CdRMgyjqtLQgp0uQ5l9bCsSv7kq1D++7Y+bMN2KvY5+f52fdcGm6hkXiLFk5A4y3M0Mm6Yt/bWpxHot3PjCYuNdo8Gr3Pda7LWguSRsO9MdXKfKF4BL9zDT2MEVrrDZfWNLJo4T1YWOSEP4xVxarTxIHojf4l6HU9ADVJkugYL4q4JQPsjJPh0vEAm3FLfsX1a1ZaU53hxiK6+FV2eMiy46N3Cbx56xfZ0SgZCFLTGHoegDDK2Cdc2LHl15+n8/V5Mmh28g7VUt7jGLKDg8fyFuVmS1dYokUt8BHy+e86m80/fru8Xi2uP10t1hdnvf8hfB76zvvILBe/sDgPje99EOgPvWzJdSG4I4UhipUx6obb05rIaD+eED+QAUl5lMAwlWEYT8MJrG5yaQprLUxV+YSDs5R3bfGZNkqiiaO9wunQ7AdZ7YwlqK2KGUsJq53yY3TpTe3fTEw41/UCMEmfWyeFbEAWJy/8SCAUd86flLVq882j6ST1+v+syvBC6s2iQav4Psi22XTeFRljk9RvX3ZSF9c/5D8

**Comment by IanLilleyT** [comment]
For now I hardcoded the camera position https://github.com/CesiumGS/cesium/commit/b3334483f6c543ce1f16a2e41d7c8c641f137567
